docs/getting-started/quick_start.md

```diff
@@ -60,31 +95,74 @@ It should be in iterable, yielding tuples, of the same length as the number of a
 
 <!--pytest-codeblocks:skip-->
 ```python
-inputset = [(2, 3), (0, 0), (1, 6), (7, 7), (7, 1)]
+max_value_included = 7
+inputset = [
+    (
+        np.random.randint(max_value_included + 1),
+        np.random.randint(max_value_included + 1),
+    )
+    for _ in range(10)
+]
 ```
 
+Here, our inputset is made of 10 pairs of integers, whose maximum value is `max_value_included`. Remark that the values which will later be used as inputs of the compiled function need to be in the same range.
+
 {% hint style="warning" %}
-All inputs in the inputset will be evaluated in the graph, which takes time. If you're experiencing long compilation times, consider providing a smaller inputset.
+All inputs in the inputset will be evaluated in the graph, which takes time. If you're experiencing long compilation times, consider providing a smaller inputset. At the same time, if the inputset is too small, bit widths may not be measured accurately, which could result in overflows during runtime, hence the result wouldn't be what's expected.
```

Comment:
Should we really have this warning here? Is a quick start example and the input set we define here should not lead to long compilation times? Should have a dedicated section in the documentation for this kind of troubleshooting?

Reply:
And maybe talk about overflows l.108, like "the input set MUST be representative of values which will be used as inputs of the compiled function else that could lead to under/overflow that have an undefined behavior during the FHE evaluation"